modules/components/containers/RuleContainer.jsx

```diff
@@ -150,13 +161,13 @@ const createRuleContainer = (Rule) =>
               onDragStart={this.props.onDragStart}
               setLock={isInDraggingTempo ? this.dummyFn : this.setLock}
               removeSelf={isInDraggingTempo ? this.dummyFn : this.removeSelf}
-              setField={isInDraggingTempo ? this.dummyFn : this.setField}
+              setField={isInDraggingTempo ? this.dummyFn : this.setField} // TODO: needed in lhs?
               setOperator={isInDraggingTempo ? this.dummyFn : this.setOperator}
               setOperatorOption={isInDraggingTempo ? this.dummyFn : this.setOperatorOption}
               setValue={isInDraggingTempo ? this.dummyFn : this.setValue}
               setValueSrc={isInDraggingTempo ? this.dummyFn : this.setValueSrc}
-              selectedField={this.props.field || null}
-              parentField={this.props.parentField || null}
+              selectedField={selectedField || null} // TODO
```

Comment:
TODO what?

Reply:
actually, this is done - `selectedField` is a field value, when field is current value source and null otherwise. The comment was just leftover, removed it.